command/hcl2_upgrade.go

```diff
@@ -266,12 +287,29 @@ func (c *HCL2UpgradeCommand) RunContext(buildCtx context.Context, cla *HCL2Upgra
 	return 0
 }
 
+type UnhandleableArgumentError struct {
+	Call           string
+	Correspondance string
+	Docs           string
+}
+
+func (uc UnhandleableArgumentError) Error() string {
+	return fmt.Sprintf(`unhandled %q call:
+# there is no way to automatically upgrade the %[1]q call.
+# Please manually upgrade to %s
+# Visit %s for more infos.`, uc.Call, uc.Correspondance, uc.Docs)
+}
+
 // transposeTemplatingCalls executes parts of blocks as go template files and replaces
 // their result with their hcl2 variant. If something goes wrong the template
 // containing the go template string is returned.
 func transposeTemplatingCalls(s []byte) []byte {
 	fallbackReturn := func(err error) []byte {
-		return append([]byte(fmt.Sprintf("\n#could not parse template for following block: %q\n", err)), s...)
+		if strings.Contains(err.Error(), "unhandled") {
+			return append([]byte(fmt.Sprintf("\n# %s\n", err)), s...)
+		}
+
+		return append([]byte(fmt.Sprintf("\n# could not parse template for following block: %q\n", err)), s...)
```

Comment:
should we ask users to open an issue if they get this error, so we can fix the command?

Reply:
Ehm, the thing is this could be caused by a wide variety of reason; one of them being that there is an error in the template and I though it'd be nice to not get issues for that 🤔 . Though one can argue that most users will try to hcl2upgrade a valid template.

Reply:
yeah I think you're right.